### PR TITLE
Use opm data

### PR DESCRIPTION
**Pull request by joakim-hove**
1. Have removed old - completely stale cmake switch `INCLUDE_NON_PUBLIC_TESTS`
2. Will look for `opm-data` and run `flow` on SPE1 case as a test.

Observe that the integration test is currently just a `it runs through with exit(0)` test.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -116,3 +119,10 @@ if (NOT EIGEN3_FOUND)
 	include_directories (${CMAKE_BINARY_DIR}/eigen3-installed/include/eigen3)
 	add_dependencies (opmautodiff Eigen3)
 endif (NOT EIGEN3_FOUND)
+
+
+
+if (HAVE_OPM_DATA)
+   add_test( NAME flow_SPE1 COMMAND flow ${OPM_DATA_ROOT}/spe1/SPE1CASE1.DATA )
+endif()   
+
```
Comment (bska) [inline CMakeLists.txt]: Just a tip, `git diff --check` frowns upon adding blank lines at EOF.

File: CMakeLists.txt [hunk 2]
```diff
@@ -116,3 +119,10 @@ if (NOT EIGEN3_FOUND)
 	include_directories (${CMAKE_BINARY_DIR}/eigen3-installed/include/eigen3)
 	add_dependencies (opmautodiff Eigen3)
 endif (NOT EIGEN3_FOUND)
+
+
+
+if (HAVE_OPM_DATA)
+   add_test( NAME flow_SPE1 COMMAND flow ${OPM_DATA_ROOT}/spe1/SPE1CASE1.DATA )
```
Comment (bska) [inline CMakeLists.txt]: Do you or don't you need the XML parmaters here?

**Comment by joakim-hove** [comment]
I try to keep the white space police happy :smiley: - amend force pushed.

File: CMakeLists.txt (hunk 2 above)
Comment (joakim-hove) [inline CMakeLists.txt]: Well - you do not need the XML parameters to run flow with all-default-parameters; so I would say no.

**Comment by bska** [comment]
> I try to keep the white space police happy :smiley: 

As a card-carrying member of that force I appreciate the effort :smirk:

**Comment by atgeirr** [comment]
Looks simple and to the point. Tested and found to work. Thanks!

I did discover an issue with Findopm-data.cmake, will submit a PR for opm-common.

**Pull request merged**